courses/free-courses-en.md

```diff
@@ -1358,6 +1358,7 @@
 * [PyTorch tutorials](https://pytorch.org/tutorials) - PyTorch.org
 * [Stanford SEE 229 - Machine Learning](https://see.stanford.edu/Course/CS229)
 * [Stanford University Machine Learning](https://www.coursera.org/learn/machine-learning)
+* [Statistics and Machine Learning in R](https://youtube.com/playlist?list=PLblh5JKOoLUJJpBNfk8_YadPwDTO2SCbx&si=KrKV9QNi-whcCsVC) - StatQuest by Josh Starmer
```

Comment:
```suggestion
* [Statistics and Machine Learning in R](https://youtube.com/playlist?list=PLblh5JKOoLUJJpBNfk8_YadPwDTO2SCbx) - StatQuest by Josh Starmer
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes